alacritty_terminal/src/grid/storage.rs

```diff
@@ -173,27 +176,29 @@ impl<T> Storage<T> {
         }
     }
 
-    /// Truncate the invisible elements from the raw buffer
+    /// Truncate the invisible elements from the raw buffer.
     pub fn truncate(&mut self) {
         self.inner.rotate_left(self.zero);
         self.inner.truncate(self.len);
 
         self.zero = 0;
     }
 
-    /// Dynamically grow the storage buffer at runtime
-    pub fn initialize(&mut self, num_rows: usize, template_row: Row<T>)
+    /// Dynamically grow the storage buffer at runtime.
+    pub fn initialize(&mut self, additional: usize, template_row: Row<T>)
```

Comment:
```suggestion
    pub fn initialize(&mut self, additional_rows: usize, template_row: Row<T>)
```

I'd assume that the name `additional` was picked because of the parameters of the `reserve` method on `Vec`, but I think it might be a bit hard to understand what `additional` means in this context. I'm not 100% sure about this, but I feel like clarifying `additional_rows` might make things a little more clear?